system-tests/projects/runner-e2e-specs/cypress/e2e/errors/docs_url.cy.js

```diff
@@ -0,0 +1,5 @@
+describe('docs url', () => {
+  it('displays doc url', () => {
+    cy.viewport()
+  })
+})
```

Comment:
[Previously](https://github.com/cypress-io/cypress/blob/10.0-release/packages/runner/cypress/fixtures/errors/docs_url_spec.js#L5), this file contained two specs that flexed between open and run mode behavior by changing the `isInteractive` config property. This felt a little weird in an e2e test, so I wrote the e2e test to flex its assertions based on the actual config value instead. But that does result in overall less coverage in CI (the open mode assertions will never run there). Any thoughts/concerns with this?

Reply:
cc @chrisbreiding ☝️

Reply:
I'd rather have tests for both, like previously, than have one test that runs differently depending on how you run it. Is there any technical hurdle to achieving that with the new test setup?

Reply:
@tbiethman I personally like the change.  System tests should not be attempting to alter cypress's internal configuration, but run like our consumer's tests would run. If there are UI changes that need to be accounted for when in run-mode, we should try to push that test to e2e or the component test-level.

Reply:
I agree with both of you 🚀 

* I don't like the magic test that changes depending on how you run it. Its coverage is inconsistent, and it'd be a pain to debug should one path break in the future. There is no technical hurdle here, the old config changes still work, at least in this case. 
* I also don't like having to change the config and disable the config validation to make this happen in an e2e test. It would make this test a lot more unique than it used to be and be less representative of user's tests.

To Emily's point, I think the best course of action (here and generally) would be to test any consistent cross-mode behavior e2e and defer any flexing logic to a component/integration/unit test. We don't have any unit tests around [this logic](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/error_utils.ts#L433) and it wouldn't be prohibitive to add some, but it's not much of a guarantee that the UI is correct. A component test for [this](https://github.com/cypress-io/cypress/blob/develop/packages/reporter/src/errors/test-error.tsx#L72-L75) would also be nice, but it might be overkill to set up component testing for the reporter package if these components are going to be changing in the near-term.

So given everything, I'm going to vote for getting the most coverage in the least amount of time and changing this back to the way it used to be, with the test-specific config changes. We will probably get to the point of needing to add component testing to the reporter anyway, and at that point I think it'd make sense to remove this test outright and use a component test instead. 👍 / 👎 ?